config/src/config.rs

```diff
@@ -461,6 +456,7 @@ impl Default for StorageConfig {
 #[derive(Debug, Deserialize, Serialize)]
 #[serde(default)]
 pub struct NetworkConfig {
+    pub peer_id: String,
```

Comment:
nit: why is `peer_id` a `String` and not a `PeerId`?

Reply:
I think default `serde` Serializer would serialize PeerId as an array of u8 integers instead of hex encoding as string, which is what we want.